application/src/main/kotlin/application/ExamplesCommand.kt

```diff
@@ -198,81 +213,165 @@ For example:
         var filterNotName: String = ""
 
         override fun call(): Int {
+            if (contractFile != null && exampleFile != null) return validateExampleFile(contractFile!!, exampleFile)
+
+            if (contractFile != null && examplesDir != null) {
+                val (exitCode, validationResults) = validateExamplesDir(contractFile!!, examplesDir)
+
+                printValidationResult(validationResults, "Example directory")
+                if (exitCode == 1) return 1
+                if (validationResults.containsFailure()) return 1
+                return 0
+            }
+
+            if (contractFile != null) return validateImplicitExamplesFrom(contractFile!!)
+
+            if (specsDir != null && examplesBaseDir != null) {
+                val exitCode = validateAllExamplesAssociatedToEachSpecIn(specsDir, examplesBaseDir)
+                return exitCode
+            }
+            if (specsDir != null) {
+                val exitCode = validateAllExamplesAssociatedToEachSpecIn(specsDir, specsDir)
+                return exitCode
+            }
+
+            logger.log("No valid options provided.")
```

Comment:
"Invalid combination of CLI options. Please refer to the help section using --help command"

Reply:
Addressed